src/main/java/org/broadinstitute/consent/http/models/AuthUser.java

```diff
@@ -69,4 +71,12 @@ public AuthUser setUserStatusInfo(UserStatusInfo userStatusInfo) {
     this.userStatusInfo = userStatusInfo;
     return this;
   }
+
+  public User getUser() {
```

Comment:
I was thinking that, in practice, a caller will know that either `User` must exist or not when this method is called. For example, `UserResource.updateSelf()` won't work unless the user exists in the database. So the goal is to `@Auth` inject an object that _must_ have the user provided. If the user isn't present, a `NotFoundException` should be thrown before its use.

My thinking is that you'd have two different `AuthUser` types. One is the one we have now, where we don't know if the duos user account exists. The other is one were it must exist, and is checked on initialization. Otherwise the caller will still need to check for a non-`null` `user` every time.